crates/hive-utils/scripts/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Compute the proxy account class hash
+export PROXY_ACCOUNT_CLASS_HASH=$(starkli class-hash ${MADARA_PATH}/cairo-contracts/kakarot/proxy.json)
```

Comment:
oh that's why we need starkli

Reply:
Indeed! Since we're not able to declare, also it will be always updated